notebooks/102-pytorch-onnx-to-openvino/102-pytorch-onnx-to-openvino.ipynb

```diff
@@ -57,7 +58,7 @@
    "source": [
     "### Settings\n",
     "\n",
-    "Set the name for the model, and the width and height of the image that will be used for the network. The CityScapes is pre-trained on images of 2048x1024 resolution. Using smaller dimensions will impact model accuracy, but will improve inference speed. "
+    "Set the name for the model, and the width and height of the image that will be used for the network. According to input transforms function, model pretrained on images with height 520 and height 780."
```

Comment:
```suggestion
    "Set a name for the model, then width and height of the image that will be used by the network during inference. According to the input transforms function, the model is pretrained on images with a height of 520 and width of 780."
```
Double check that what I changed is correct @eaidova .. height was repeated twice at the end